lib/templates/lib/rubyvernac/language_alias_loader.rb

```diff
@@ -0,0 +1,40 @@
+require 'yaml'
+
+class LanguageAliasLoader
```

Comment:
Is this class going inside the language gem or staying core? Preference would be to keep this in core-gem.

Reply:
I've moved this logic into core gem.